editor.planx.uk/src/routes/published.tsx

```diff
@@ -25,7 +27,15 @@ const routes = compose(
 
   mount({
     "/": route({
-      view: <Questions previewEnvironment="standalone" />,
+      view: () => {
+        const isOnline = useStore.getState().flowStatus === "online";
+
+        return isOnline ? (
+          <Questions previewEnvironment="standalone" />
+        ) : (
+          <OfflinePage />
+        );
+      },
```

Comment:
Nice & simple :+1:

Reply:
Took longer than I care to admit to whittle it down to this 😅